titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/jobmanager/client/GrpcJobManagementClient.java

```diff
@@ -213,12 +213,13 @@ public Completable killTask(TaskKillRequest taskKillRequest) {
         }, configuration.getRequestTimeout());
     }
 
-    private void reportErrorMetrics(Set<ValidationError> errors) {
+    private void reportErrorMetrics(Set<ValidationError> errors, JobDescriptor jobDescriptor) {
         errors.forEach(error ->
                 registry.counter(
                         error.getField(),
                         "type", error.getType().name(),
-                        "description", error.getDescription())
+                        "description", error.getDescription(),
```

Comment:
Should this be bounded? I am not sure if there are limits on metric values in Atlas.
Is it guaranteed to be always non-null?

Reply:
Atlas bounds it.  It's a required field in all constructors.  A user of `ValidationError` would have to explicitly set it to `null`.